src/libPersistence/Retriever.cpp

```diff
@@ -167,11 +168,12 @@ bool Retriever::RetrieveTxBlocks(bool trimIncompletedBlocks) {
     }
   }
 
-  if (boost::filesystem::exists("StateDeltaFromS3")) {
+  if (boost::filesystem::exists(STORAGE_PATH + "/StateDeltaFromS3")) {
```

Comment:
Can the constant `STATEDELTAFROMS3_PATH` be used here?

Reply:
Fixed